osu.Game/Tests/Visual/OsuManualInputManagerTestScene.cs

```diff
@@ -50,9 +51,13 @@ protected OsuManualInputManagerTestScene()
             {
                 var cursorDisplay = new GlobalCursorDisplay { RelativeSizeAxes = Axes.Both };
 
-                cursorDisplay.Add(content = new OsuTooltipContainer(cursorDisplay.MenuCursor)
+                cursorDisplay.Add(new PopoverContainer
                 {
                     RelativeSizeAxes = Axes.Both,
+                    Child = content = new OsuTooltipContainer(cursorDisplay.MenuCursor)
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                    },
```

Comment:
This change is intentionally grouped away from the regressing test provided. It aims to match the `OsuGameBase` usage in a local way we can actually access from the test (to check the popover did display).